doc/compe.txt

```diff
@@ -184,15 +345,13 @@ VIMSCRIPT CONFIG                                           *compe-usage-ex-viml*
   let g:compe.source.buffer = v:true
   let g:compe.source.calc = v:true
   let g:compe.source.vsnip = v:true
+  let g:compe.source.lamp = v:true
```

Comment:
Should lamp also be in examples, or was it omitted previously on purpose?

Reply:
yeah, it was omitted on purpose and I think it'll get removed in the future